request.el

```diff
@@ -67,6 +67,10 @@
   "Executable for curl command."
   :type 'string)
 
+(defcustom request-curl-options nil
+  "curl command options."
```

Comment:
@syohex maybe this should be expanded to explain how it works? Something like "List of strings that will be passed to every curl invocation. You can pass extra options here, like setting the proxy."